client/build.gradle

```diff
@@ -8,11 +8,6 @@ node {
     download = true
 }
 
-task bootRun(dependsOn: 'start') {
-    group = 'application'
-    description = 'Run the client app (for use with gradle bootRun -parallel'
-}
-
```

Comment:
Can we add this back in or was it breaking something?
not having it breaks the
gradle bootRun -parallel